### PR TITLE
Type Conversions

### PR DESCRIPTION
**Pull request by arthur-gyulabyan**
Translated article of "Type Conversion" section.

### CODE REVIEW

**Comment by bugron** [comment]
Thanks for the PR. I'll review it today.

File: 1-js/02-first-steps/07-type-conversions/article.md [hunk 1]
```diff
@@ -1,150 +1,150 @@
...
 
-Explicit conversion is usually required when we read a value from a string-based source like a text form but expect a number to be entered.
+Բացահայտ փոխարկումը հաճախ նախընտրելի է, երբ որևէ թվի մուտքագրում ակնկալող տեքստային դաշտից արժեք վերցնելու կարիք կա։
 
-If the string is not a valid number, the result of such a conversion is `NaN`. For instance:
+Երբ տողից թվի փոխարկումը վավեր չէ, այդ փոխարկման արդյունքը կլինի `NaN`։ Օրինակ․
 
 ```js run
-let age = Number("an arbitrary string instead of a number");
+let age = Number("կամայակն տող, թվի փոխարեն");
```
Comment (bugron) [inline 1-js/02-first-steps/07-type-conversions/article.md]:
```suggestion
let age = Number("կամայական տող, թվի փոխարեն");
```

File: 1-js/02-first-steps/07-type-conversions/article.md [hunk 2]
```diff
@@ -1,150 +1,150 @@
...
+Առավել հաճախ կիրառվող տիպային փոխարկումները տողային, թվային և տրամաբանական փոխարկումներն են։
 
-**`String Conversion`** -- Occurs when we output something. Can be performed with `String(value)`. The conversion to string is usually obvious for primitive values.
+**`Տողային փոխարկում`** -- Տեղի է ունենում ինչ-որ բանի պատկերման դեպքում։ Կարող է իրականացվել `String(value)`-ի միջոցով։ Տողային փոխարկումը պրիմիտիվ արժեքների դեպքում կանխատեսելի է։
 
-**`Numeric Conversion`** -- Occurs in math operations. Can be performed with `Number(value)`.
+**`Թվային փոխարկում`** -- Տեղի է ունենում մաթեմատիկական գործողություններում։ Կարող է իրականացվել `Number(value)`-ի միջոցով։
 
-The conversion follows the rules:
+Փոխարկումը ենթարկվուվ է հետևյալ կանոններին․
```
Comment (bugron) [inline 1-js/02-first-steps/07-type-conversions/article.md]:
```suggestion
Փոխարկումը ենթարկվում է հետևյալ կանոններին․
```

File: 1-js/02-first-steps/07-type-conversions/article.md [hunk 3]
```diff
@@ -1,150 +1,150 @@
...
 |`null`|`0`|
 |<code>true&nbsp;/&nbsp;false</code> | `1 / 0` |
-| `string` | The string is read "as is", whitespaces from both sides are ignored. An empty string becomes `0`. An error gives `NaN`. |
+| `string` | Եզրերում եղած բացատները անտեսվում են։ Դատարկ տողը դառնում է `0`։ Ձախողման դեպքում ստացվում է `NaN`։ |
 
-**`Boolean Conversion`** -- Occurs in logical operations. Can be performed with `Boolean(value)`.
+**`Տրամաբանական փոխարկում`** -- Տեղի է ունենում տրամաբանական գործողություններում։ Կարող է իրականացվել `Boolean(value)`-ի միջոցով։
 
-Follows the rules:
+Ենթարկվուվ է հետևյալ կանուններին․
```
Comment (bugron) [inline 1-js/02-first-steps/07-type-conversions/article.md]:
```suggestion
Ենթարկվում է հետևյալ կանուններին․
```

**Review by bugron — CHANGES_REQUESTED** [review]
Just a few typos to fix otherwise looks great!

**Comment by arthur-gyulabyan** [comment]
Thanks for the review.

**Review by bugron — APPROVED** [review]
LGTM

**Pull request merged**